pkg/engine/utils.go

```diff
@@ -142,7 +149,7 @@ func matchSubjects(ruleSubjects []rbacv1.Subject, userInfo authenticationv1.User
 	return false
 }
 
-func doesSliceContainsAnyOfTheseValues(slice []string, values ...string) bool {
+func DoesSliceContainsAnyOfTheseValues(slice []string, values ...string) bool {
```

Comment:
Should we move this to a utility package so we can reuse without a dependency on engine?